#7: Material Design: Toolbar / App bar

joergreichert opened the pull request:
(https://github.com/Polymap4/polymap4-rhei/issues/issue/7)

first version of toolbar consisting of navigation menu, label and button
composite

joergreichert committed:
#7: Material Design: Toolbar / App bar
(https://github.com/Polymap4/polymap4-rhei/issues/issue/7)

first version of toolbar consisting of navigation menu, label and button
composite

fb71 commented:
Warum benutzen wir hier nicht den SWT ToolBar? Das sollte gut dokumentiert sein. Wie ist der Stand/Status? Was fehlt? Wie kann das fehlende implementiert werden? Was sind offene Probleme?

Achte bitte auch bei meinen Änderungen darauf, dass die Dokumentation ordentlich ist. Andere Programmierer müssen verstehen warum hier was gemacht wurde - auch wenn man dich nicht direkt fragen kann. Das ist sehr wichtig. Ansonsten wird die Entwicklung mit der Anzahl der Zeilen Code immer langsamer und schlechter, egal wieviele Leute wir einstellen.

joergreichert committed:
#7: Material Design: Toolbar / App bar
(https://github.com/Polymap4/polymap4-rhei/issues/issue/7)

Some polishing of toolbar, added some documentation
